### DISCUSSION TITLE
Error running provided sample code -->> bad magic [got 0x67676d66 want 0x67676a74]

### DISCUSSION THREAD

**Discussion by feuerchop** [nomic-ai/pygpt4all#65] [Q&A]
Hi - hope someone can help with this. I followed the instruction and downloaded gpt4all-lora-quantized-ggml.bin from the site. Then I tried to run the sample code, however I got following error: 

> llama_model_load: loading model from '../xxxx/models/gpt4all-lora-quantized-ggml.bin' - please wait ...
../xxxx/models/gpt4all-lora-quantized-ggml.bin: invalid model file (bad magic [got 0x67676d66 want 0x67676a74])
        you most likely need to regenerate your ggml files
        the benefit is you'll get 10-100x faster load times
        see https://github.com/ggerganov/llama.cpp/issues/91
        use convert-pth-to-ggml.py to regenerate from original pth
        use migrate-ggml-2023-03-30-pr613.py if you deleted originals
llama_init_from_file: failed to load model
llama_generate: seed = 1681884033
system_info: n_threads = 8 / 12 | AVX = 1 | AVX2 = 1 | AVX512 = 0 | FMA = 1 | NEON = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | VSX = 0 | 
[1]    77594 segmentation fault  ....

Can you advice how to fix this?

**Comment by absadiki** [comment]
@feuerchop, I think the model you have downloaded is corrupted, or not converted to the new format! 
From where have you downloaded it ?

**Reply by feuerchop** [reply]
I downloaded it from here: 
https://the-eye.eu/public/AI/models/nomic-ai/gpt4all/

**Reply by ElenaKutanov** [reply]
The same here:
https://the-eye.eu/public/AI/models/nomic-ai/gpt4all/gpt4all-lora-quantized-ggml.bin

ERROR: ./models/gpt4all-lora-quantized-ggml.bin: invalid model file (bad magic [got 0x67676d66 want 0x67676a74]) ...

**Reply by absadiki** [reply]
You can find all the models here: https://github.com/nomic-ai/gpt4all-chat#manual-download-of-models
